### PR TITLE
(Bug #1008) Add landing page for /file area

### PR DESCRIPTION
**Pull request by hotlevel4**
Provides links to /file/new and /file/edit. Does not provide a link to
/file/index as that does not appear to be in use. Fixes issue #1008.

### CODE REVIEW

File: cgi-bin/DW/Controller/Media.pm [hunk 1]
```diff
@@ -31,6 +31,7 @@ DW::Routing->register_regex( qr!^/file/(\d+x\d+|full)(/\w:[\d\w]+)*/(\d+)$!,
 DW::Routing->register_string( '/file/list', \&media_manage_handler, app => 1 );
 DW::Routing->register_string( '/file/edit', \&media_bulkedit_handler, app => 1 );
 DW::Routing->register_string( '/file/new', \&media_new_handler, app => 1  );
+DW::Routing->register_string( '/file/', \&media_index_handler, app => 1 );
```
Comment (zorkian) [inline cgi-bin/DW/Controller/Media.pm]: There should not be a trailing slash on this. For example:

```
Shop.pm:DW::Routing->register_string( '/shop', \&shop_index_handler, app => 1 );
Shop.pm:DW::Routing->register_string( '/shop/points', \&shop_points_handler, app => 1 );
```

**Comment by zorkian** [comment]
Happy to merge after that trailing slash is removed!

**Comment by hotlevel4** [comment]
Fixed!

**Comment by zorkian** [comment]
Thanks!

**Pull request merged**